spec/lib/account_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'spec_helper'
+
+describe BankAccounts::Account do
+
+  describe "class methods" do
+    it "responds to 'new'" do
+      expect(BankAccounts::Account).to respond_to :new
+    end
+
+    let(:account) do
+      BankAccounts::Account.new("Voltron", 1000)
+    end
+
+      it "subtracts withdraw(amount) and returns updated balance" do
+        expect(account.withdraw(50)).to eq(950.00)
+      end
+
+    context "when withdraw(amount) would result in a negative value of balance" do
+      let(:account) do
```

Comment:
You shouldn't need to redefine `account` here. It will inherit it from the parent context